tests/js/server/aql/aql-modify-cluster.js

```diff
@@ -1730,6 +1728,73 @@ function ahuacatlUpdateSuite () {
   };
 }
 
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+function ClusterLimitSuite () {
```

Comment:
I'd like to suggest some additional tests for this suite.

- First, tests for replace as well as the existing insert/update/remove.
- Tests that use an offset in the `LIMIT`
- Tests that additionally use `{ fullCount: true }`

And finally, what I mentioned yesterday. As the `FILTER` currently overfetches, I think something like this will still break:
```
FOR doc IN ${cn1}
  FILTER doc.valueMod < 10
  REMOVE doc IN ${cn1}
  FILTER OLD.valueMode < 10
  LIMIT ${modify_docs}
  RETURN 0
```
If this is supposed to work - @jsteemann, can you confirm that it should - the `FILTER` block must fetch `atMost` rather than `DefaultBlockSize`. Plus, this would have to be fixed in 3.4, too!

And, if that is the case, we should have tests for it as well.